src/components/Sidebar/NestedTableOfContentsSidebar.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import classnames from 'classnames';
+
+import Sidebar from './Sidebar';
+import { SidebarSection } from './Section';
+import { Parser } from "html-to-react"
+
+const NestedTableOfContentsSidebar = ({ tableOfContents, className }) => {
+  const htmlToReactParser = new Parser();
+
+  const reactComponents = htmlToReactParser.parse(tableOfContents);
```

Comment:
There's actually a simpler way to do this. Instead of HTML -> React -> HTML, you can just dump the raw HTML into the page. https://reactjs.org/docs/dom-elements.html#dangerouslysetinnerhtml

This is not actually dangerous in this instance, since the HTML we are inserting is not end-user generated.

```
<div className="table-of-contents-sidebar" dangerouslySetInnerHTML={{ __html: tableOfContents }} />
```

Reply:
oh right cool. That means I dont need that yarn lock change. cheers